lib/katello/repo_discovery.rb

```diff
@@ -0,0 +1,3 @@
+Dir["#{File.expand_path('../../../app/lib/katello/resources/discovery', __FILE__)}/*.rb"].each do |file|
```

Comment:
You don't need `expand_path`, but I wonder why you even need this `load` in the first place. I think it's very much incompatible with the Zeitwerk autoloader.

Reply:
That was because of what you said in the review below around naming the modules correctly. It loads now without this explicit loading.